src/Sylius/Component/Attribute/Model/Attribute.php

```diff
@@ -170,4 +170,12 @@ public function getStorageType()
     {
         return $this->storageType;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isValueTranslatable()
+    {
+        return AttributeValueInterface::STORAGE_TEXT == $this->getStorageType() ? true : false;
```

Comment:
fixed useless IF inline :+1: 
STORAGE_TEXT is used also for textarea (and it should be used for any additional text attribute, I suppose)